src/gossip/fuzz_service.zig

```diff
@@ -358,17 +421,17 @@ pub fn run(args: *std.process.ArgIterator) !void {
                 const packet = packets.items[0];
                 break :blk packet;
             },
-            4 => blk: {
+            Actions.pull_response => blk: {
```

Comment:
Any reason you qualified these? Seems somewhat redundant, but open to hearing your rationale.

Reply:
eb28c8e

mb